packages/lib/src/index.ts

```diff
@@ -132,7 +130,7 @@ export type {
 } from './interactions/models';
 export { MouseButton } from './interactions/models';
 
-export type { Domain, Dims, Axis } from '@h5web/shared';
+export type { Domain, VisibleDomains, Dims, Axis } from '@h5web/shared';
```

Comment:
I extract the return type of `getVisibleDomains` into `@h5web/shared`, next to `Domain`.